include/modem/at_monitor.h

```diff
@@ -67,12 +97,12 @@ void at_monitor_init(void);
  * @param ... Optional monitor initial state (@c PAUSED or @c ACTIVE).
  *	      The default initial state of a monitor is active.
  */
-#define AT_MONITOR(name, _filter, _handler, ...)                               \
-	static void _handler(const char *);                                    \
-	STRUCT_SECTION_ITERABLE(at_monitor_entry, at_monitor_##name) = {       \
-		.filter = _filter,                                             \
-		.handler = _handler,                                           \
-		COND_CODE_1(__VA_ARGS__, (.paused = __VA_ARGS__,), ())         \
+#define AT_MONITOR_ISR(name, _filter, _handler, ...)                                               \
+	static void _handler(const char *);                                                        \
+	STRUCT_SECTION_ITERABLE(at_monitor_isr_entry, at_monitor_isr_##name) = {                   \
+		.filter = _filter,                                                                 \
+		.handler = _handler,                                                               \
+		COND_CODE_1(__VA_ARGS__, (.paused = __VA_ARGS__,), ())                             \
 	}
 
 /**
```

Comment:
Below we'll have `at_monitor_pause` and `at_monitor_resume`, which won't work for ISR monitors. I've made copies for ISR in my SMS PR context but not sure if there is a better way.

Reply:
Why would those not work for ISR monitors? 🤔

Reply:
Aaah the type name is different :)
Let me fix that

Reply:
@trantanen fixed